### PR TITLE
Change some subsections into sections (Three essential tools)

### PR DESCRIPTION
**Pull request by martinmoene**
Aim for a table of contents that reads as an overview of the book.

### CODE REVIEW

**Comment by martinmoene** [comment]
I'd like to suggest to change the chapter title from:
>The three most essential tools

to

>Three essential tools

*"It doesn't make sense"* ( ;-)  ) to use *most* in combination with *essential*.
Something is essential or it is not, it doesn't allow for ranking.

**Comment by grzesiek-galezowski** [comment]
Agree on "most essential" - please add it to the pull request.
By the way, I agree it might make sense to change "xUnit framework" to "Test framework", although I think it forces making a change to the next line, which says: "The first and most essential tool we are going to use is an xUnit framework.".

We can apply the "most essential" fix here as well and write it as something like:
"The first of the essential tools we are going to use is a test framework - a tool for running our tests."

What do you think?

**Comment by martinmoene** [comment]
Yeah, I already noticed the sentence would have to change, but postponed that (untill now).

.... thinking ....

**Comment by martinmoene** [comment]
How about (something like):
*The first essential tool we'll use is a test framework. A test framework allows us to specify and execute our tests.*

**Comment by grzesiek-galezowski** [comment]
looks good. Please add it to the pull request and I'll merge it.

**Comment by martinmoene** [comment]
Done.

File: manuscript/040_Three_Essential_Tools.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-The three most essential tools
-==============================
+The essential tools
```
Comment (martinmoene) [inline manuscript/040_Three_Essential_Tools.md]: mmm, shouldn't this be *Three essential tools* ?

**Pull request merged**